packages/lib/src/components/ThreeDS2/components/utils.ts

```diff
@@ -207,3 +237,11 @@ export const get3DS2FlowProps = (actionSubtype, props) => {
         i18n: props.i18n
     };
 };
+
+export const isInIframe = () => {
+    try {
```

Comment:
There is no need for a try/catch here, just need to check window is defined and then return the boolean expression.

Reply:
As discussed over mattermost, window.top can actually throw an exception, this is a security feature and it will throw an exception if the origin is not the same as parent, however that means this code will always return false. So it should return true in the catch I think. but in general I wonder if there is a cleaner way to do this..🤔